src/extensions/install/install.ts

```diff
@@ -9,26 +8,43 @@ import { DEFAULT_REGISTRY_DOMAIN_PREFIX } from '../../constants';
 export class Install {
   constructor(private workspace: Workspace, private packageManager: PackageManager) {}
   async install() {
-    try {
-      const components = await this.workspace.list();
-      const isolatedEnvs = await this.workspace.load(components.map(c => c.id.toString()));
-      const packageManagerName = this.workspace.consumer.config.packageManager;
+    const components = await this.workspace.list();
+    const isolatedEnvs = await this.workspace.load(components.map(c => c.id.toString()));
+    const packageManagerName = this.workspace.consumer.config.packageManager;
+    await Promise.all(
+      isolatedEnvs.map(async e => {
+        const componentPackageName = componentIdToPackageName(
+          e.component.id.legacyComponentId,
+          DEFAULT_REGISTRY_DOMAIN_PREFIX
+        );
+        try {
+          await fs.unlink(path.join(process.cwd(), 'node_modules', componentPackageName));
+        } catch (err) {
+          // if the symlink does not exist - no problem
+          if (err.code !== 'ENOENT') {
+            throw err;
```

Comment:
faster to check if exist instead of try catch (the catching is much slower then checking existence)

Reply:
Checking existence is not atomic though...